test/yup.js

```diff
@@ -7,7 +7,7 @@ import { object, array, string, lazy, number } from '../src';
 describe('Yup', function(){
 
   it('should export', function(){
-    require('../lib') // eslint-disable-line global-require
+    require('../src') // eslint-disable-line global-require
```

Comment:
this doesn't test the same thing. This is supposed to be a smoke test for the final compiled module not being broken.

Reply:
Should we remove this test then?